clippy_lints/src/doc/mod.rs

```diff
@@ -422,6 +424,39 @@ declare_clippy_lint! {
     "require every line of a paragraph to be indented and marked"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    /// Detects doc comment linebreaks that use double spaces to separate lines, instead of back-slash (\).
+    ///
+    /// ### Why is this bad?
+    /// Double spaces, when used as doc comment linebreaks, can be difficult to see, and may
+    /// accidentally be removed during automatic formatting or manual refactoring. The use of a back-slash (\)
+    /// is clearer in this regard.
+    ///
+    /// ### Example
+    /// The two replacement dots in this example represent a double space.
+    /// ```no_run
+    /// /// This command takes two numbers as inputs and··
+    /// /// adds them together, and then returns the result.
+    /// fn add(l: i32, r: i32) -> i32 {
+    ///     l + r
+    /// }
+    /// ```
+    ///
+    /// Use instead:
+    /// ```no_run
+    /// /// This command takes two numbers as inputs and \
```

Comment:
Should probably not have a space, see also https://github.com/rust-lang/rust-clippy/pull/12876#discussion_r1623250888
```suggestion
    /// /// This command takes two numbers as inputs and\
```

Reply:
@Jacherr, I think this is still open, or did you intentionally keep it this way?

Reply:
Yep, somehow missed this 😅. Should be resolved now.